.github/workflows/build.yml

```
@@ -40,6 +40,17 @@ jobs:
       with:
         python-version: ${{ env.PYTHON_VERSION }}
 
+    - name: Manual Installation Steps (macOS)
+      if: ${{ runner.os == 'macOS' }}
+      run: |
+        sudo rm -rf /Libary/Developer/CommandLineTools
+        echo A | softwareupdate --install-rosetta
+        arch -x86_64 xcode-select --install
+        arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.github.usercontent.com/Homebrew/install/master/install.sh)"
+        export PATH="/usr/local/opt/node@16/bin:/usr/local/bin:$PATH"
+        arch -x76_64 brew install node@16 yarn git python@$PYTHON_VERSION
```

There's an error in CI about not being able to install Rosetta 2.

(Rosetta being the translation layer to run x86_64 code on ARM (Apple Silicon), that makes sense.)

We _presumably_ don't need the Rosetta or `arch` command stuff, but I can confirm `arch -86_64 [some command here]` works on my intel Mac. And doing the least changes possible from the Cirrus code seems fair for a first go to prove whether this works.

Suggested diff is to not try to install Rosetta, and also I noticed a typo a few lines below that (`-x76_64` --> `x86_64`).

```suggestion
        arch -x86_64 xcode-select --install
        arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.github.usercontent.com/Homebrew/install/master/install.sh)"
        export PATH="/usr/local/opt/node@16/bin:/usr/local/bin:$PATH"
        arch -x86_64 brew install node@16 yarn git python@$PYTHON_VERSION
```

Very curious how this turns out. Is using NodeJS/python/yarn from homebrew the fix all along??? (????????) Maybe it's just having homebrew on the system in general? IDK. Worth trying!

---

I am too slow.

---

Thanks for the comment, seems you are totally right.

I'll just keep going down the trail of install commands, until it finally stops complaining.

And who knows, if that's the fix, that'll be amazing

---

Oh, and copying from the _intel_ Mac task definition on Cirrus should be ideal the place to copy from, rather than the Apple Silicon task definition. That's what I did earlier today to try and intel macOS build on Cirrus, and it worked 👍.